### PR TITLE
Remove <link> usage

### PR DESCRIPTION
**Pull request by jameswex**
Add wit-dashboard invocation to wit_jupyter.html so that colab and jupyter implementations can remove use of the <link> HTML import, as this is removed in upcoming Chrome M80 and due to this, witwidget never worked in Firefox and similar browsers.

With this change, and a new pip version release, witwidget will work in Chrome M80 and Firefox/Safari.

### CODE REVIEW

File: witwidget/notebook/colab/wit.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def compute_custom_distance(wit_id, index, callback_name, params):
     (function() {{
       const id = {id};
       const wit = document.querySelector("#wit");
-      wit.parentElement.style.height = '{height}px';
+      wit.style.height = '{height}px';
```
Comment (tolga-b) [inline witwidget/notebook/colab/wit.py]: why was this change necessary?

File: witwidget/notebook/jupyter/js/lib/wit.js [hunk 2]
```diff
@@ -57,18 +57,10 @@ var WITView = widgets.DOMWidgetView.extend({
       witHtmlLocation = window.__nbextension_path__ + 'wit_jupyter.html';
     }
 
-    const src = `<link rel="import" href="${witHtmlLocation}">
-    <wit-dashboard local id="wit"></wit-dashboard>
```
Comment (tolga-b) [inline witwidget/notebook/jupyter/js/lib/wit.js]: I think this change is related to the one in wit.html. Can you just comment on what this line is moved to wit.html?

File: witwidget/notebook/colab/wit.py (hunk 1 above)
Comment (jameswex) [reply witwidget/notebook/colab/wit.py]: discussed offline, due to how wit.html contains WIT

File: witwidget/notebook/jupyter/js/lib/wit.js (hunk 2 above)
Comment (jameswex) [reply witwidget/notebook/jupyter/js/lib/wit.js]: discussed offline
